### PR TITLE
Assignment-8 Class Components

### PR DESCRIPTION
**Pull request by roshantrivedi**

### CODE REVIEW

**Review by ap221882 — COMMENTED** [review]
Reviewed. Kudos.

File: React_Day8/avengers/src/ClassComponents/GrandParentClass.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+class GrandParentClass extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { color: "RED" };
+    console.log("inside grand-parent class constructor()");
+  }
+
+  async componentDidMount() {
+    console.log("Grand-Parent componentDidMount()  ");
```
Comment (nitishnivedan) [inline React_Day8/avengers/src/ClassComponents/GrandParentClass.js:12]: not making any api call. so async is not needed

File: React_Day8/avengers/src/ClassComponents/ParentClass.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+  componentDidUpdate() {
+    console.log("Parent componentDidUpdate() ");
+  }
+
+  render() {
+    return (
+      <>
+        <h2>Exploring Class Components (II)</h2>
+        {console.log("inside parent class render()")}
```
Comment (nitishnivedan) [inline React_Day8/avengers/src/ClassComponents/ParentClass.js:25]: this console.log, better not to write it here. you can mention this console between render and return

File: React_Day8/avengers/src/index.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
+import ReactDOM from 'react-dom/client';
+import './index.css';
```
Comment (nitishnivedan) [inline React_Day8/avengers/src/index.js:2]: can give an empty line between library imports and local file imports for better readibility

**Pull request merged**